.circleci/config.yml

```diff
@@ -607,13 +592,16 @@ jobs:
             --validators-in-genesis $PREMAINNET_NUM_VAL \
             --seed $PREMAINNET_CONFIG_SEED
       - run:
-          name: Premainnet cluster test
+          name: Run diagnostics
           command: |
-            cargo run -p cluster-test -- --swarm --emit-tx --mint-file /tmp/config/mint.key --peers $NODES
+            .circleci/diagnostics.sh .circleci/nodes.csv /tmp/message.txt
+            export failed=$?
+            export message=$(cat /tmp/message.txt)
+          working_directory: .circleci
       - slack/status:
           fail_only: true
           webhook: "${WEBHOOK_PREMAINNET}"
-          failure_message: "<@channel> Cluster test failed on premainnet :ferris-detective:"
+          failure_message: "${message}"
```

Comment:
This is the only part of the pr I'm not sure of -- should work - can test in CI

Reply:
ah yes, got too wrapped up in the script, less thought in to circle ci integration.